keps/sig-instrumentation/3498-extending-stability/README.md

```diff
@@ -311,19 +343,18 @@ This should not affect upgrade/rollback paths.
 
 ###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
 
-`Alpha` metrics will be recategorized as `Internal`.
+No.
 
 ### Monitoring Requirements
 
 ###### How can an operator determine if the feature is in use by workloads?
 
-You can determine this by seeing if workloads depend on any Kubernetes control-plane metrics. If they do, they are using this feature.
+We've introduced a metric (i.e. `registered_metrics_total`) which should serve to indicate this feature is enabled.
```

Comment:
L367 - let's switch to "No" - there aren't any things that we would like to add that aren't added, right?